fixed default link & added appropriate links to resources #144

sridhareaswaran opened the pull request:
**Worked on:**
Issue #144 - Close down / archive / redirect old doc

**Changes:**
1. Removed default URL in `card.html`
2. Added appropriate URLs to resources listed in `books.md`

sridhareaswaran committed:
fixed default link & added appropriate links to resources

Commit:
Added useful browser extension details under Tools

mlvandijk commented on content/tools/general.md:
```diff
@@ -25,6 +25,10 @@ It offers several [packages](https://atom.io/packages/search?q=cucumber) you can
 [TextMate](https://macromates.com/) is a text editor for macOS.
 See the [`Cucumber.tmbundle`](https://github.com/cucumber/cucumber-tmbundle) documentation.
 
+## Tidy Gherkin
```
This tool should be on the [Related Tools](http://docs.cucumber.io/tools/related-tools/) page instead of the general tools page. Could you move it there please?

sridhareaswaran replied:
Sure, I will move that.

mlvandijk commented on themes/cucumber-hugo/layouts/shortcodes/card.html:
```diff
@@ -1,4 +1,4 @@
-<a href="http://google.com">
+<a href="{{ .Get "link" }}">
   <div class="card" style="height: 100%">
```
Going to a specific link per book instead of Google home page is a great idea. Just not sure about what the particular links should be

mlvandijk commented on content/professional/books.md:
```diff
@@ -11,23 +11,26 @@ We like these Books
          title="Discovery"
-         image="/img/book-discovery.png"
+         image="/img/book-discovery.png" 
+         link="https://www.goodreads.com/book/show/36126762-the-bdd-books"
```
I like having individual links per book, rather than go to google home page. Not sure it should go to GoodReads though. @aslakhellesoy @mattwynne @sebrose - do you have an opinion on this?

sridhareaswaran replied:
Ya, I felt the same. But then the reason for choosing GoodReads is that it provides genuine user reviews, multiple vendors and formats (Physical, digital & audio) for the user to choose from. Let's hear from others too.

mattwynne replied:
From an author's point of view, linking to the Pragmatic Programmers' page for the books that are published through them means we get a better royalty payment than if you link to Amazon. Otherwise, just do what seems most useful to the reader.

mlvandijk reviewed (changes requested):
Thank you for offering improvements! Just a question for the Cucumber people regarding where to link from the books (see below) and a request to move the tool to a different tools-page. 

Also, I'm not sure how this is related to #144 which is about cleaning up the old links from the wiki..

sridhareaswaran committed:
moving Tidy Gherkin info to the Related Tools page

moving Tidy Gherkin info to the [Related Tools](http://docs.cucumber.io/tools/related-tools/) page.
